mne_bids/tests/test_mne_bids.py

```diff
@@ -219,6 +232,9 @@ def test_bti():
     cmd = ['bids-validator', output_path]
     run_subprocess(cmd, shell=shell)
 
+    raw, events, _ = read_raw_bids(bids_basename + '_meg',
+                                   output_path)
```

Comment:
everything is broken :-P

when we copy the `.pdf` files, we don't retain the `config` files. Is this intentional @teonbrooks ?